spec/context/dimension.fmf

```diff
@@ -0,0 +1,50 @@
+summary: Supported context dimensions
+
+story:
+    As a tester I want to clearly specify dimensions, such as
+    'distro' or 'trigger', so that I can well describe the
+    context in which a test is running.
+
+description: |
+    The following dimensions are supported:
+
+    product
```

Comment:
IMHO `product` is just for internal usage - where we expect to have distro != product (e.g. rhscl-3.5 on rhel-7). Isn't it always the same for Fedora and Centos?

Reply:
I see, I thought the software collections are available for centos as well. Removing.

Reply:
There are software collections for CentOS! 

https://www.softwarecollections.org/en/scls/

Reply:
But are they a separate product in CentOS? IMHO no.

Reply:
Does Centos Have any "products"? _I don't know. Maybe? There are SIGs, which care-take some portions of the content (built on top of CentOS 7). SIGs usually care take about anything that's not in core CentOS - it's not built using any automation (apart from what SIGs create). The SIG created content is also less "restricted" WRT builds- specifically, AFAIK you can submit any SRPM for build - no repo means no commit hooks, no build checks (gating) that I know of... Also there can be any content that's not in RHEL (community created SCLs)._

What I know, is: you need to explicitly install/enable some repositories (using dnf install) to be able to install the SCLs.

Reply:
I'd suggest to go on with `distro` only for now and add the `product` if/when there is a clear need for it, hm?